web/pdf_page_view.js

```diff
@@ -879,7 +879,6 @@ class PDFPageView {
       this.textLayer = new TextLayerBuilder({
         highlighter: this._textHighlighter,
         accessibilityManager: this._accessibilityManager,
-        isOffscreenCanvasSupported: this.isOffscreenCanvasSupported,
```

Comment:
Given that this option is now unused, please remove *all* traces of it from this file such that we don't ship "dead" code.

*Note:* This applies to the `web/pdf_viewer.js` file too, as well as [here](https://github.com/mozilla/pdf.js/blob/dc01782633432642fe8b56e3c25bdc9a430d9e02/web/app.js#L444).